src/coreComponents/finiteElement/FiniteElementDispatch.hpp

```diff
@@ -57,23 +58,27 @@ dispatch3D( FiniteElementBase const & input,
   {
     lambda( *ptr4 );
   }
-#ifdef GEOSX_DISPATCH_VEM
-  else if( auto const * const ptr5 = dynamic_cast< H1_Tetrahedron_VEM_Gauss1 const * >(&input) ) // VEM on Tetrahedron
+  else if( auto const * const ptr5 = dynamic_cast< Q3_Hexahedron_Lagrange_GaussLobatto const * >(&input) )
   {
     lambda( *ptr5 );
   }
-  else if( auto const * const ptr6 = dynamic_cast< H1_Pyramid_VEM_Gauss1 const * >(&input) ) // VEM on Pyramid
+#ifdef GEOSX_DISPATCH_VEM
+  else if( auto const * const ptr6 = dynamic_cast< H1_Tetrahedron_VEM_Gauss1 const * >(&input) ) // VEM on Tetrahedron
   {
     lambda( *ptr6 );
   }
   else if( auto const * const ptr7 = dynamic_cast< H1_Wedge_VEM_Gauss1 const * >(&input) ) // VEM on Wedge
   {
     lambda( *ptr7 );
   }
-  else if( auto const * const ptr8 = dynamic_cast< H1_Hexahedron_VEM_Gauss1 const * >(&input) ) // VEM on Hexahedron
+  else if( auto const * const ptr8 = dynamic_cast< H1_Wedge_VEM_Gauss1 const * >(&input) ) // VEM on Prism / Wedge
```

Comment:
Isn't this "else if" a duplicate of the previous one (line 70)? Is it intended? Same at line 124